barretenberg/cpp/src/barretenberg/translator_vm/translator_proving_key.cpp

```diff
@@ -1,105 +1,69 @@
 #include "translator_proving_key.hpp"
 namespace bb {
-
 /**
- * @brief Compute new polynomials which are the concatenated versions of other polynomials
+ * @brief Construct a set of polynomials that are the result of interleaving a group of polynomials. Used in translator
+ * to reduce the degree of the permutation relation.
  *
- * @details Multilinear PCS allow to provide openings for concatenated polynomials in an easy way by combining
- * commitments. This method creates concatenated version of polynomials we won't need to commit to. Used in Goblin
- * Translator
+ * @details Multilinear PCS allow to provide openings for the resulting interleaved polynomials without having to commit
+ * to them, using the commitments of polynomials in groups.
  *
- * Concatenation in Translator mean the action of constructing a new Polynomial from existing ones by writing
- * their multilinear representations sequentially. For example, if we have f(x₁,x₂)={0, 1, 0, 1} and
- * g(x₁,x₂)={1, 0, 0, 1} then h(x₁ ,x₂ ,x₃ )=concatenation(f(x₁,x₂),g(x₁,x₂))={0, 1, 0, 1, 1, 0, 0, 1}
+ * If we have f(x₁,x₂)={0, 1, 0, 1} and g(x₁,x₂)={1, 0, 0, 1} then
```

Comment:
haha I think maybe this example with 0's and 1's was a bit more clear when it was pure concatenation but with interleaving it would probably be much more clear to just use something more general like `a_i`/`b_i`

Reply:
truee